.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,51 @@
+name: presubmit
+on: [push, pull_request]
+jobs:
+  check:
+    name: Source checks
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install clang-format
+      - name: C++ style check
```

Comment:
Am I missing something, or are we missing a Python format check?  I think the old rules were missing it too

Reply:
Yes, it wasn't enforced before. I can add it.

Reply:
Done.